Formula/frps.rb

```diff
@@ -0,0 +1,73 @@
+class Frps < Formula
+  desc "This is the server of frp (https://github.com/fatedier/frp)"
```

Comment:
This URL is already below, no need to have it twice.

Reply:
remove duplicated url from desc.  on commit 7d6d68d